From 6bdba673a1f2f03a211403cd7865ab1c7e737eaa Mon Sep 17 00:00:00 2001
From: Ashwin Dayanand Kamat <kashwindayan@vmware.com>
Date: Wed, 2 Nov 2022 14:38:23 +0000
Subject: [PATCH] ebtables: Initialise len and flags in ebt_nflog_info
 structure

The modern compilers might do this by default.
But for safe side and to have clean code, the below patch has been
added as per HCX Team requirement.
---
 extensions/ebt_nflog.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/extensions/ebt_nflog.c b/extensions/ebt_nflog.c
index 04c547d..d3bbf96 100644
--- a/extensions/ebt_nflog.c
+++ b/extensions/ebt_nflog.c
@@ -50,6 +50,8 @@ static void init(struct ebt_entry_watcher *watcher)
 	struct ebt_nflog_info *info = (struct ebt_nflog_info *)watcher->data;
 
 	info->prefix[0] = '\0';
+	info->len = 0;
+	info->flags = 0;
 	info->group = EBT_NFLOG_DEFAULT_GROUP;
 	info->threshold = EBT_NFLOG_DEFAULT_THRESHOLD;
 }
-- 
2.23.3

